src/types/io.ts

```diff
@@ -241,8 +241,12 @@ export type AoWeightedObserver = {
 export type AoGatewayWeights = {
   stakeWeight: number;
   tenureWeight: number;
+  // @deprecated - use `gatewayPerformanceRatio` instead
   gatewayRewardRatioWeight: number;
+  // @deprecated - use `observerPerformanceRatio` instead
   observerRewardRatioWeight: number;
+  gatewayPerformanceRatio: number;
+  observerPerformanceRatio: number;
```

Comment:
in the method, we could duplicated this deprecated value to prevent any backwards compatiblity issues. at least until release 4.0.0

Reply:
the contract will continue to provide these values for now - then we can prep the update pre-mainnet and likely release a v4 release with all the final outputs